packages/google-closure-compiler/lib/grunt/index.js

```diff
@@ -218,6 +224,42 @@ module.exports = (grunt, pluginOptions) => {
       });
   }
 
+  // 
+  /**
+   * Grabs `ps` as array of promise-returning functions and `done` function as callback.
+   * Separates ps` into batches of length == compileInBatches and runs resulting 
+   * promises in batches in parallel but batches in series.
+   * 
+   * @param {Array<Function(...?):Promise>}} functions returning promises
+   * @param {Function(null)} callback for the end of processing
```

Comment:
```suggestion
   * @param {function():undefined} done callback for the end of processing
```

Reply:
Done. And then saw "Commit suggestion" button :)